traffic_ops/traffic_ops_golang/routes.go

```diff
@@ -162,15 +162,15 @@ func Routes(d ServerData) ([]Route, []RawRoute, http.Handler, error) {
 		//User: CRUD (1.4 then 1.1)
 		//Incrementing version for users because change to Nullable struct. Delete is completely new.
 		{1.4, http.MethodGet, `users/?(\.json)?$`, api.ReadHandler(user.GetTypeSingleton()), auth.PrivLevelReadOnly, Authenticated, nil},
-		{1.4, http.MethodGet, `users/{id}?(\.json)?$`, api.ReadHandler(user.GetTypeSingleton()), auth.PrivLevelReadOnly, Authenticated, nil},
-		{1.4, http.MethodPut, `users/{id}?(\.json)?$`, api.UpdateHandler(user.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
+		{1.4, http.MethodGet, `users/{id}$`, api.ReadHandler(user.GetTypeSingleton()), auth.PrivLevelReadOnly, Authenticated, nil},
+		{1.4, http.MethodPut, `users/{id}$`, api.UpdateHandler(user.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
 		{1.4, http.MethodPost, `users/?(\.json)?$`, api.CreateHandler(user.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
-		//{1.4, http.MethodDelete, `users/{id}?(\.json)?$`, api.DeleteHandler(user.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
+		//{1.4, http.MethodDelete, `users/{id}$`, api.DeleteHandler(user.GetTypeSingleton()), auth.PrivLevelOperations, Authenticated, nil},
```

Comment:
When I first put it in we were considering adding an implementation of some sort. I remember there being a github issue and possibly something discussed on the apache dev list. It seems that it might be awhile before we actually do something like that. There might be a comment about it someplace else in the code.

I normally wouldn't keep commented out code, but this is a single line and tells the viewer fairly explicitly "we don't have a delete for this endpoint".

Reply:
It's more confusing than informative IMO. When I saw that I thought there was actually a Golang implementation but it wasn't finished or something. In this case there is just a no-op handler to satisfy the CRUDer interface.

Reply:
Well, that's kinda what it was. I made it be a no-op handler just in case someone thought "hey, I'll just uncomment this". That would have been a bad idea since the generic delete would not have been aware of tenancy or priv_level.